### ISSUE TITLE
Rename gRPC API bundles for clarity

### ISSUE THREAD

**Issue by wtlucy** [OpenLiberty/open-liberty#19247]
Currently Liberty provides a `io.openliberty.io.grpc.1.0_{version}.jar` as a stable API, and `io.openliberty.grpc.client.1.0.thirdparty_{version}.jar` as a third-party API. These API bundles don't actually provide any `1.0` gRPC version - so it's been proposed that we should remove the `1.0` completely from the names, making them just ~`io.openliberty.io.grpc.1.0_{version}.jar`~ `io.openliberty.io.grpc_{version}.jar` and `io.openliberty.grpc.client.thirdparty_{version}.jar`.

**Comment by pnicolucci** [comment]
@wtlucy in the above comment you say the following :

> These API bundles don't actually provide any 1.0 gRPC version - so it's been proposed that we should remove the 1.0 completely from the names, making them just io.openliberty.io.grpc.1.0_{version}.jar and io.openliberty.grpc.client.thirdparty_{version}.jar.

Did you mean `io.openliberty.io.grpc_(version).jar` and not `io.openliberty.io.grpc.1.0_{version}.jar` just making sure I understand the ask here.

**Comment by wtlucy** [comment]
Good catch @pnicolucci - I intended to write `io.openliberty.io.grpc_{version}.jar`. I'll update the description.
